src/Renderer/RendererOpenGL.cpp

```diff
@@ -94,7 +94,7 @@ RendererOpenGL::RendererOpenGL(const std::string& title) : Renderer(title)
  */
 RendererOpenGL::~RendererOpenGL()
 {
-	Utility<EventHandler>::get().windowResized().disconnect(this, &RendererOpenGL::resizeViewport);
+	Utility<EventHandler>::get().windowResized().disconnect(this, &RendererOpenGL::resizeWindow);
```

Comment:
This commit is labelled as renaming `_resize` to `resizeWindow`, which makes this change look out of place.